lib/active_encode/file_handler.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+module ActiveEncode
+  module FileHandler
```

Comment:
I like how these methods have been encapsulated, but I wonder if they would more logically live under the `FfmpegAdapter`.  Maybe the `#remove_old_files!` method could be pulled out into this module as well and mixed into the adapter.  Then it could maybe have a name like `ActiveEncode::EngineAdapters::FfmpegAdapter::Cleaner`.  Do you think that makes sense or did you think these methods might get used elsewhere in ActiveEncode?

Reply:
That makes sense. I had originally been thinking about it potentially being used elsewhere, but looking over things I think the only other place where it might be relevant as written is the pass through adapter. And it will remove those files anyways since passthrough uses the same WORK_DIR and file naming as the ffmpeg adapter.